curricula/views.py

```diff
@@ -1009,6 +1009,19 @@ def unit_element(request, unit_name, format=None):
     serializer = UnitSerializer(unit)
     return Response(serializer.data)
 
+
+@api_view(['GET', ])
+def unit_export(request, unit_name, format=None):
+    try:
+        unit = Unit.objects.get(login_required=False, status=2, unit_name=unit_name)
+    except MultipleObjectsReturned:
```

Comment:
Should we do anything for no objects returned? Or are we okay with that just failing?

Reply:
good idea! it looks like there is a helper for that. before:

![Screen Shot 2020-11-05 at 2 55 10 PM](https://user-images.githubusercontent.com/8001765/98305663-2824c600-1f77-11eb-81e6-7838e6dba8c9.png)

after:

![Screen Shot 2020-11-05 at 2 54 24 PM](https://user-images.githubusercontent.com/8001765/98305676-2d821080-1f77-11eb-9a16-45753250b50c.png)

docs: https://docs.djangoproject.com/en/1.8/topics/http/shortcuts/#get-object-or-404

Reply:
I will fix up the code studio script to give some idea that this 404 has happened in this case.